AdCOM v1.0 FINAL.md

```diff
@@ -4036,6 +4039,64 @@ The following table lists the types of feeds for audio.
 </table>
 
 
+### List:  IFA Types <a name="list_ifa_types"></a>
+
+The following table lists the source types of the device IFA.
```

Comment:
Calling out here as well that this list doesn't correspond to previous IAB guidance, nor reflects the full universe of platform-provided IFA types that may be encountered in the wild.

CC @hillslatt @ChrisBasis @jwilhite-ias @iantri @emailcurt

Reply:
Specific IDs were removed per a previous Supply Chain Working Group [proposal](https://docs.google.com/document/d/1ko5l88-sS-7HC7TZJW_BwvCAk9f7gzLsFXRp1QjHimM), and were reintroduced for this one.

Reply:
Resolving conversation since ifa was pushed to a later release pending larger conversation around buy-side id signaling (eids, ifa, and buyeruid)